schema/setup/control_50_servers_cache.cql

```diff
@@ -0,0 +1,14 @@
+USE @@KEYSPACE@@;
+
+CREATE TABLE servers_cache (
+    tenant_id ascii,
+    group_id ascii,
+    last_fetch timestamp,
+    server_id ascii,
+    server_blob ascii,
+    PRIMARY KEY((tenant_id, group_id), last_fetch, server_id)
+) WITH CLUSTERING ORDER BY (last_fetch DESC, server_id ASC) AND 
```

Comment:
Not really, I could remove `server_id ASC` and CASS will still keep it sorted. I just added it for completeness. The important thing is `last_fetch DESC` since that is how the data is going to be stored and how the query is going to be made.